src/background/stores/global/scoping-store.ts

```diff
@@ -41,7 +44,12 @@ export class ScopingStore extends BaseStoreImpl<ScopingStoreData> {
     private onAddSelector = (payload: ScopingPayload): void => {
         let shouldUpdate: boolean = true;
         _.forEach(Object.keys(this.state.selectors[payload.inputType]), key => {
-            if (_.isEqual(this.state.selectors[payload.inputType][key], payload.selector)) {
+            if (
```

Comment:
This strikes me as another case where the "vertical expansion" isn't particularly helpful